src/memory_service/fam_memory_service_direct.cpp

```diff
@@ -450,6 +451,13 @@ Fam_Memory_Service_Direct::get_config_info(std::string filename) {
             // If parameter is not present, then set the default.
             options["fam_path"] = (char *)strdup("");
         }
+        try {
```

Comment:
Add it into memory service config file also.

Reply:
Not clear. 
---
} catch (Fam_InvalidOption_Exception e) {
            // If parameter is not present, then set the default.
            options["garbage_collector_threads"] = (char *)strdup("0");
---
this is what we have now. Did you mean the comment to be added ?